### PR TITLE
Refactored code

### PR DESCRIPTION
**Pull request by trong-le**

### CODE REVIEW

File: test/test-config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default  {
+	apiKey: process.env.API_KEY,
```
Comment (perryao) [inline test/test-config.js]: indentation

File: lib/index.js [hunk 2]
```diff
@@ -36,49 +32,10 @@ export default class TypeFormSource extends Activity {
...
 
-        return new rp(options)
-            .then((response) => {
-                return response;
-            })
-            .catch((err) => {
-                return err;
-            })
+        const response = await rp(options);
+        arr.push(response.questions);
```
Comment (perryao) [inline lib/index.js]: why not just return the response directly?

File: test/test.js [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
 });
 
-test('oauth onTaskCopy', async (t) => {
-	const source = new TypeFormSource();
-	const data = await source.onTaskCopy(config);
-});
+test('test api', async t => {
+    const activity = new TypeFormSource();
+    const data = await activity.getTypeformData(config);
+    const questionAnswersArray = activity.getResponses(data);
```
Comment (perryao) [inline test/test.js]: anything you can await here? and getResponses has been removed.

File: lib/index.js (hunk 2 above)
Comment (trong-le) [inline lib/index.js]: When I tried doing that I got the error `[2016-10-26 16:46:43,228] {docker_operator.py:164} INFO - ERROR - Error executing task: Error: Object was not a stream, promise, iterator or iterable: object`

Comment (perryao) [inline lib/index.js]: what is the type of response.questions? It's plural, so I'd assume it's an array.

**Pull request merged**